packages/webapp/src/assets/colors.scss

```diff
@@ -69,6 +69,8 @@
   --tableSortPill: #f4f5f7;
 
   // New design system colours
+  --White: #fff;
```

Comment:
Do we need this one? lol . Its a very minor thing but I wonder if instead of `var(--White)` we can use `white`

Reply:
I think the only advantage is being able to re-define it all at once throughout app if we want, right? If Loïc ever did want to make this an off-white or something? (Although I'll be we have so much hard-coded through app anyway 😢)

Reply:
Yeah for sure and that makes sense to me.  Maybe then it is worth a specific name (or names) that would make it more re-definable? Like `--secondary-btn-background: #fff` . I don't think I would redefine white haha